Options lock-* were removed (#28920)
hashicorp/terraform: website/docs/cli/commands/init.html.md

* `-input=true` Ask for input if necessary. If false, will error if
  input was required.

> * `-lock=false` Disable locking of state files during state-related operations
> 
>     -> Note: This option was removed in Terraform 0.15.

Reviewer: Thank you for calling this out! I did two things. First, I added the note into the bullet text. This way, it doesn't interrupt the flow of the page as much and users are much more likely to see/read it when they are reading the actual description of the command. I also added a link to where we explain more about why we removed this/what it did. 

```suggestion
* `-lock=false` ([removed in Terraform 0.15](/upgrade-guides/0-15.html#other-minor-command-line-behavior-changes)) Disable locking of state files during state-related operations.
```

> * `-lock-timeout=<duration>` Override the time Terraform will wait to acquire

Reviewer: same as above! 

```suggestion
* `-lock-timeout=<duration>` ([removed in Terraform 0.15](/upgrade-guides/0-15.html#other-minor-command-line-behavior-changes)) Override the time Terraform will wait to acquire
```

  a state lock. The default is `0s` (zero seconds), which causes immediate
  failure if the lock is already held by another process. (Removed since version 0.15.0)

>     -> Note: This option was removed in Terraform 0.15.
>     

Reviewer: ```suggestion
```
